50-tools/weblate-functions.inc

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+
+set -o errexit
+
+function weblate_django_shell() {
+	if test -z "$(which weblate-ssh)" ; then
+		echo "This command needs ssh access to l10n.opensuse.org server." >&2
+		echo "Please install/create weblate-ssh script wrapper." >&2
```

Comment:
Question: Do I need this tool? Where do I get it from?

Reply:
@sknorr We could contact the author [Stanislav Brabec](https://github.com/stanislav-brabec) for explanations about it. What do you think?

Reply:
@stanislav-brabec What is the purpose of the whole weblate-create-projects script? Where would I get the weblate-ssh tool from?